publisher/publisher.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
 	"time"
 
+	common "github.com/Azure/azure-amqp-common-go/v3"
```

Comment:
hmm, is that ok? I'm really just using a dead simple retry... autorest has one too. thoughts?

Reply:
You're not getting much from it you wouldn't get from avast. Your biggest risk is that they change how they decide when to retry but Presuabley they'd change common.Retryable at that point.